.github/workflows/moodle-release.yml

```diff
@@ -1,64 +1,20 @@
 #
-# Whenever a new tag starting with "v" is pushed, add the tagged version
+# Whenever version.php is changed, add the latest version
 # to the Moodle Plugins directory at https://moodle.org/plugins
 #
 name: Releasing in the Plugins directory
 
 on:
   push:
-    tags:
-      - '*'
-
-  workflow_dispatch:
-    inputs:
-      tag:
-        description: 'Tag to be released'
-        required: true
-
-defaults:
-  run:
-    shell: bash
+    paths:
+      - 'version.php'
 
 jobs:
-  release-at-moodle-org:
-    runs-on: ubuntu-latest
-    env:
-      PLUGIN: auth_enrolkey
-      CURL: curl -s -u ${{ secrets.CURL_AUTH }}
-      ENDPOINT: https://next.moodle.org/webservice/rest/server.php
-      TOKEN: ${{ secrets.MOODLE_ORG_TOKEN }}
-      FUNCTION: local_plugins_add_version
-
-    steps:
-      - name: Call the service function
-        id: add-version
-        run: |
-          if [[ ! -z "${{ github.event.inputs.tag }}" ]]; then
-            TAGNAME="${{ github.event.inputs.tag }}"
-          elif [[ $GITHUB_REF = refs/tags/* ]]; then
-            TAGNAME="${GITHUB_REF##*/}"
-          fi
-          if [[ -z "${TAGNAME}" ]]; then
-            echo "No tag name has been provided!"
-            exit 1
-          fi
-          ZIPURL="https://api.github.com/repos/${{ github.repository }}/zipball/${TAGNAME}"
-          RESPONSE=$(${CURL} ${ENDPOINT} --data-urlencode "wstoken=${TOKEN}" \
-                                         --data-urlencode "wsfunction=${FUNCTION}" \
-                                         --data-urlencode "moodlewsrestformat=json" \
-                                         --data-urlencode "frankenstyle=${PLUGIN}" \
-                                         --data-urlencode "zipurl=${ZIPURL}" \
-                                         --data-urlencode "vcssystem=git" \
-                                         --data-urlencode "vcsrepositoryurl=${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}" \
-                                         --data-urlencode "vcstag=${TAGNAME}" \
-                                         --data-urlencode "changelogurl=${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/commits/${TAGNAME}" \
-                                         --data-urlencode "altdownloadurl=${ZIPURL}")
-          echo "::set-output name=response::${RESPONSE}"
-
-      - name: Evaluate the response
-        id: evaluate-response
-        env:
-          RESPONSE: ${{ steps.add-version.outputs.response }}
-        run: |
-          jq <<< ${RESPONSE}
-          jq --exit-status ".id" <<< ${RESPONSE} > /dev/null
+  workflow_group_35_plus_release:
+    uses: catalyst/catalyst-moodle-workflows/.github/workflows/group-35-plus-release.yml@main
```

Comment:
this references workflow touches a different set of stables here:

https://github.com/catalyst/catalyst-moodle-workflows/blob/main/.github/workflows/group-35-plus-release.yml#L65-L72

So do we need the same combinations of branches strategy for handling releasing as well? ie should we have a group-35-to_39-release.yml@main file instead?

Reply:
The release file does only support pgsql database. So I removed the combinations for mariadb. The same I can see over here https://github.com/danmarsden/moodle-mod_attendance/blob/MOODLE_311_STABLE/.github/workflows/moodle-release.yml#L36

Reply:
Hey @brendanheywood - I've added another condition for plugin release - https://github.com/catalyst/catalyst-moodle-workflows/blob/main/.github/workflows/group-35-plus-release.yml#L79

So the plugin release step only works for "MOODLE_311_STABLE" job. 

Also I released new version using this branch in plugin directory - https://moodle.org/plugins/auth_enrolkey/versions 

In order to avoid this release, added branch "master", so release workflow will trigger on push or pull request but only for the master branch
- https://github.com/catalyst/moodle-auth_enrolkey/pull/90/files#diff-eb27b680e545edb5cf1bdf78d64bd754b135159acde486aa84a384f1f4a55fd4R10

Is that fine?

Thanks,
Anu